mycity/mycity/intents/get_alerts_intent.py

```diff
@@ -77,27 +77,69 @@ def get_alerts_intent(
     :param get_alerts_function_for_test: Injectable function for unit tests
     :param prune_normal_responses_function_for_test: Injectable function
      for unit tests
-    :param alerts_to_speech_output_function_for_test: Injectable function 
+    :param alerts_to_speech_output_function_for_test: Injectable function
     for unit tests
     :return: MyCityResponseDataModel object
     """
     logger.debug('MyCityRequestDataModel received:' +
                  mycity_request.get_logger_string())
 
-    alerts = get_alerts() if get_alerts_function_for_test is None \
-        else get_alerts_function_for_test()
-    logger.debug("[dictionary with alerts scraped from boston.gov]:\n" +
-                 str(alerts))
+    # get the intent_variables and sessions_attribute object from the request
+    intent_variables = mycity_request.intent_variables
+    session_attributes = mycity_request.session_attributes
+    decision = intent_variables['Decision'].get('value') \
+        if 'Decision' in intent_variables else None
+    session_alerts = session_attributes.get('alerts')
```

Comment:
Not sure if this is an improvement or not, so will leave it up to you: We may be able to clean up the code below if we always make sure we have the list of alerts. All the code below can make the assumption we have a valid list. We could change this line to be something like:
`session_alerts = session_attributes.get('alerts', get_pruned_alerts())`
where `get_pruned_alerts()` is lines 105-114 below